pkg/client/src/app/queries/applications.ts

```diff
@@ -125,3 +126,31 @@ export const useDeleteApplicationMutation = (
     },
   });
 };
+
+export const useBulkDeleteApplicationMutation = (
+  onSuccess: () => void,
+  onError: (err: AxiosError) => void,
+  getApplicationAssessment?: (id: number) => Assessment | undefined
```

Comment:
It would be nice to remove the associated assessments when removing the applications as we do for single app delete.
https://github.com/konveyor/tackle2-ui/pull/235

Reply:
@gildub Have you tested that the cascading delete on the API side works without the UI code? Otherwise we will still need this implemented.

Reply:
Sounds like @jortel plans on adding the getApplicationsAssessment to the hub. So you can remove line 133.